region/norfair/east/Lower Norfair Elevator.json

```
@@ -688,58 +671,180 @@
     },
     {
       "id": 31,
-      "link": [3, 1],
+      "link": [4, 1],
       "name": "Base",
       "requires": [
-        {"heatFrames": 30}
+        {"heatFrames": 40}
```

```suggestion
        {"heatFrames": 50}
```
I don't think 40 is possible. 44 is the best I could get. 50 will match the Carry Shinecharge strat. There's an asymmetry here (based on where Samus' shot spawns?), where technically you could put 45 for the left door and 50 for the right door. I've just been doing them symmetrically using the conservative value that works for both doors, with the idea that this way these won't need modification in case we ever want to support horizontal mirroring.

I believe that for the in-game times shown in the Practice Hack here, the timer starts when the elevator ride ends, not when first entering the room; I notice I get the same results whether fast elevators option is enabled or not.